src/tools/types/BlockToolDataScheme.ts

```diff
@@ -0,0 +1,7 @@
+import { DataType } from './DataType';
+import { DataKey } from '../../entities';
+
+/**
+ * Block Tool data scheme to describe the structure of the data
+ */
+export type BlockToolDataScheme = Record<DataKey, DataType>;
```

Comment:
What if Tool contains TextNodes not on the first level, like NestedList or Table tool:

```
data: {
  items: [
    {
      content: ''
      items: [...]
    }
  ]
}
```